app/src/components/ArticleDetails.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { createStyles, WithStyles } from '@material-ui/styles';
+
+import Img from 'react-image';
+import {
+    Typography,
+    Box,
+} from '@material-ui/core';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { Link } from 'react-router-dom';
+import bigInt from 'big-integer';
+import { mdiEmoticonHappyOutline, mdiEmoticonNeutralOutline, mdiEmoticonSadOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import stc from 'string-to-color';
+import classNames from 'classnames';
+import { Styles } from '../styles/ArticleDetails';
+import {
+    Article,
+} from '../models/NewsTimeline';
+import { NANOSECONDS_IN_ONE_SECOND } from '../constants/Constants';
+import {
+    getHoursMinutesStringFromTimestamp,
+    getDateStringFromTimestamp,
+} from '../utils/date-utils';
+import { Colors } from '../styles/App';
+
+const styles = createStyles(Styles);
+
+interface Props extends WithStyles<typeof styles>{
+    article: Article,
+    instruments: Array<string>,
+}
+
+interface State {}
+
+class ArticleDetails extends Component<Props, State> {
+    render() {
+        const { article, classes, instruments } = this.props;
+
+        const nanosecondTimestamp = bigInt(article.timestamp).multiply(NANOSECONDS_IN_ONE_SECOND);
+        const timePublishedString = getHoursMinutesStringFromTimestamp(nanosecondTimestamp);
+        const datePublishedString = getDateStringFromTimestamp(nanosecondTimestamp);
+        const orderedInstruments = article.tickers.sort(a => {
+            return instruments.includes(a) ? 0 : 1;
+        });
```

Comment:
@chimano since you sort on the article's tickers already at parent level, this sort isn't necessary i think

Reply:
Ah i hadn't noticed that repeat